integration/test/Test/Swagger.hs

```diff
@@ -127,3 +129,21 @@ testSwaggerLint sw = do
         liftIO $ putStrLn out
         liftIO $ putStrLn err
         assertFailure "swagger validation errors"
+
+testRenderAllSwaggers :: (HasCallStack) => App ()
+testRenderAllSwaggers = do
+  tempdir <- liftIO $ createTempDirectory "/tmp/" "wire-swagger-versions"
+  putStrLn $ "testRenderAllSwaggers: writing to " <> tempdir
+  let writeSwagger :: String -> Response -> App ()
+      writeSwagger filename resp = do
+        content <- show <$> resp.json
+        liftIO $ writeFile (tempdir </> filename <> ".json") content
+
+  for_ existingVersions $ \v -> do
+    bindResponse (BrigP.getSwaggerPublicAllJson v) $ \resp -> do
+      resp.status `shouldMatchInt` 200
+      writeSwagger (show v) resp
+  for_ (Set.toList internalApis) $ \a -> do
+    bindResponse (BrigP.getSwaggerInternalJson a) $ \resp -> do
+      resp.status `shouldMatchInt` 200
+      writeSwagger a resp
```

Comment:
This is not a real test, let's not add things in integration tests just because we need to execute stuff. But let's discuss this in #4367, which wouldn't block the release?